gen3-client/g3cmd/generate-tsv.go

```diff
@@ -1,110 +0,0 @@
-package g3cmd
-
-import (
-	"crypto/md5"
-	"encoding/csv"
-	"encoding/hex"
-	"fmt"
-	"io"
-	"log"
-	"os"
-	"path/filepath"
-
-	"github.com/spf13/cobra"
-	"github.com/uc-cdis/gen3-client/gen3-client/logs"
-)
-
-func computeMD5(file string) string {
-	h := md5.New()
-	f, err := os.Open(file)
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer f.Close()
-	if _, err := io.Copy(h, f); err != nil {
-		log.Fatal(err)
-	}
-	return hex.EncodeToString(h.Sum(nil))
-}
-
-func init() {
-	var template string
-	var output string
-
-	var generateTSVCmd = &cobra.Command{
```

Comment:
I would hope we can remove it, just because it is a feature that seldomly get used

But I can revisit and try to add it back for now

Reply:
might want to ask user services and bios if they use it?

Reply:
I don't think either user serivces or bios think this cmd will still be useful to them. But just to make it less disruptive instead of deleting it straightly, I'll mark it as `deprecated` and leave a dummy command in here. User service will update the wordings on gen3.org accordingly

@paulineribeyre can I get another review? 😛